src/tooling/docs-assembler/assembler.yml

```
@@ -32,6 +32,7 @@ environments:
 
 named_git_references:
   stack: &stack 9.0
```

This could because something like 
```suggestion
  stack: &stack
    current: 9.0
    next: 9.1
```

And then you only do 

```yaml
beats: *stack
```

---

I couldn't get something like that to work locally. Does this suggestion work for you?

---

You're right. It doesn't work for me as well.. Although I think it should. 

Probably some quick on how we desrialize the config file.